target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf286d.dts

```diff
@@ -0,0 +1,462 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2022, Pawel Dembicki <paweldembicki@gmail.com>.
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "ZTE MF286D";
+	compatible = "zte,mf286d";
+
+	reserved-memory {
+		#address-cells = <0x01>;
+		#size-cells = <0x01>;
+		ranges;
+
+		rsvd1@87000000 {
+			reg = <0x87000000 0x500000>;
+			no-map;
+		};
+
+		wifi_dump@87500000 {
+			reg = <0x87500000 0x600000>;
+			no-map;
+		};
+
+		rsvd2@87b00000 {
+			reg = <0x87b00000 0x500000>;
+			no-map;
+		};
+	};
+
+	aliases {
+		led-boot = &led_internal;
+		led-failsafe = &led_internal;
+		led-running = &led_internal;
+		led-upgrade = &led_internal;
+	};
+
+	chosen {
+		bootargs-append = " root=/dev/ubiblock0_1";
```

Comment:
Hm, openwrt does autodetection on this. Does uboot already append some sort of "root=" that you are trying to overwrite? Because it be swell, if you could drop the bootargs-append here and rename the rootfs to "ubi" so you don't need that CI_UBIPART="rootfs" in the sysupgrade platform.sh.

Reply:
Now bootargs looks that:
`[    0.000000] Kernel command line: ubi.mtd=rootfs root=mtd:ubi_rootfs rootfstype=squashfs rootwait root=/dev/ubiblock0_1`
It won't boot without mangle with `root=` string.

Reply:
Ok. Can you please add something like "uboot passes: 'ubi.mtd=rootfs root=mtd:ubi_rootfs rootfstype=squashfs rootwait' " as a comment here. 

No need to change/drop bootargs-append then.

Reply:
Done.